src/assets/brand/favicon.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <defs>
+    <style>
+      .c-favicon {
+        fill: #215cca;
+      }
+      @media (prefers-color-scheme: light) {
+        .c-favicon {
+          fill: #215cca;
+        }
+      }
+      @media (prefers-color-scheme: dark) {
+        .c-favicon {
+          fill: #fff;
+        }
+      }
```

Comment:
This isn't a component, so we probably prefix its class with `c-`?

Reply:
@tylersticka 

> Do you think there's anything we can do to make the icon work better in browsers where light mode is selected but the inactive tabs are darker?
> 
> <img alt="Screen Shot 2020-08-17 at 4 28 01 PM" width="414" src="https://user-images.githubusercontent.com/69633/90453897-a7a4a680-e0a6-11ea-91d3-b153b79d0b9f.png">

That is an excellent question. The first thing that comes to mind is to explore using `mix-blend-mode`. I will test this out and as well as review other possible solutions and their outcomes.

Here's a non-favicon example of `mix-blend-mode: luminosity;` 

![mix-blend-mode](https://user-images.githubusercontent.com/1427548/90575823-26661600-e171-11ea-9a88-b29b6814f74a.gif)